src/utils/jss/ShadowRootPortal.tsx

```diff
@@ -16,65 +14,82 @@ globalThis.getComputedStyle = new Proxy(
     globalThis?.window?.getComputedStyle || globalThis.getComputedStyle || (() => {}),
     {
         apply(target, thisArg, args) {
-            if (args[0] === proxy) args[0] = document.body
+            // ! getComputedStyle won't work with proxy
+            if (!(args[0] instanceof Element)) args[0] = document.body
             return Reflect.apply(target, thisArg, args)
         },
     },
 )
 Object.assign(globalThis.window || {}, { getComputedStyle })
 
-let proxy: HTMLElement | undefined
+let proxy: ShadowRoot | undefined
+const handler: ProxyHandler<ShadowRoot> = {
+    // ! (1) to make it more like a Document
+    // ! (2) to make it more like an Element
+    get(target, property, handler) {
+        const ours = ['namespaceURI']
```

Comment:
Why list this attr here? It's not supported by Chrome now https://developer.mozilla.org/en-US/docs/Web/API/Node/namespaceURI

Reply:
react or something tried to read this so I put it here...